consonance-webservice/pom.xml

```diff
@@ -419,7 +419,17 @@
             <artifactId>swagger-core</artifactId>
             <version>${swagger.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-models</artifactId>
+            <version>${swagger.version}</version>
+            <!--<version>1.5.9</version>-->
```

Comment:
Small comment, I rarely check in already commented code unless its extremely clear that I will use it/implement it down the road, which doesn't seem to be the case here.